install/build-docker-images.sh

```diff
@@ -3,8 +3,8 @@ echo "${_group}Building and tagging Docker images ..."
 echo ""
 # Build any service that provides the image sentry-self-hosted-local first,
 # as it is used as the base image for sentry-cleanup-self-hosted-local.
-$dc build --build-arg "http_proxy=${http_proxy:-}" --build-arg "https_proxy=${https_proxy:-}" --build-arg "no_proxy=${no_proxy:-}" --force-rm web
-$dc build --build-arg "http_proxy=${http_proxy:-}" --build-arg "https_proxy=${https_proxy:-}" --build-arg "no_proxy=${no_proxy:-}" --force-rm
```

Comment:
Doesn't this break current users' workflow using `http_proxy` env variable?

Reply:
Yes you are right. But I think it's very specific scenario: such users can access the Dockerhub without any proxy, but need http proxy for some resources during Sentry installation, like https://github.com/getsentry/wal2json.

For users who access any Internet resources (including Dockerhub) during installation it's required to set HTTP Proxy several times:
1. Docker systemd file for `docker pull`
2. `~/.docker/config.json` for any commands inside docker containers
3. `/etc/environment` for settings shell proxy variables

I think we can set only do 1. and 2. But in this case `docker build` will fail because in the commands above we explicitly overwrite proxy variables with empty values, that is why I suggest to modify these lines.

Actually we can keep these lines unchanged but from our discussion https://github.com/getsentry/develop/pull/622#discussion_r912366290 I'd found that this behavior is a bit unclear, that is why I'd created this PR with such changes.

Reply:
Let me check it one more time. I'd tried to install Sentry with my changes again and there are some issues.

Reply:
I'd rechecked this one more time. Proxies from `~/.docker/config.json` work only for `docker run` command and do not work for `docker build`. It is written in the [docker docs](https://docs.docker.com/network/proxy/):

> When you create or start new containers, the environment variables are set automatically within the container.

So we have to pass proxy env variables explicitly for the build command, that is why this PR shouldn't be merged.